inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```

Comment:
Why not use numpy's .std function, instead of manually writing another implementation?

Reply:
In fact, I think this is calculating the variance, instead of the standard deviation.